src/test/java/org/folio/calendar/integration/api/health/OkapiHealthTest.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.calendar.integration.api.health;
+
+import static org.hamcrest.Matchers.is;
+
+import org.folio.calendar.integration.BaseApiTest;
+import org.folio.calendar.integration.ValidationSchema;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+/**
+ * A good example of a test class within the application.
+ * Note that the endpoint being tested is not implemented in any controller but instead built into Spring Boot.
+ *
+ * The only Okapi requirement for /admin/health is to respond with a 200 status code.
+ * @see https://wiki.folio.org/display/DD/Back+End+Module+Health+Check+Protocol
+ */
+class OkapiHealthTest extends BaseApiTest {
+
+  public static final String HEALTH_API_ROUTE = "/admin/health/";
```

Comment:
Mostly wanted to keep it consistent with the other API tests

Reply:
I do tend to value concise expressiveness over consistency, so I am a bit biased. 

I tend to believe many constants are redundant and too broadly scoped (I think this might be a hangover from languages where it was expected that these would be class or global level).

Reply:
Totally get it -- I've removed this constant.